client/src/Components/HomePage.js

```diff
@@ -1,42 +1,46 @@
-import React from 'react';
-import { useNavigate } from 'react-router-dom';
-import Navbar from './Navbar';
-import data from '../Assests/HomePageData';
-import CategoryCard from './CategoryCard';
+import {React,useState} from 'react'
+import Navbar from './Navbar'
+import {useNavigate} from 'react-router-dom'
+import data from '../Assests/HomePageData'
+import CategoryCard from './CategoryCard'
+import ProductCard from './ProductCard'
+import Products from './Products'
```

Comment:
_Why_ did you import products component when we aren't even using it in this HomePage.js file?